plugins/Facebook/class.facebook.plugin.php

```diff
@@ -13,7 +13,7 @@ class FacebookPlugin extends Gdn_Plugin {
     const API_VERSION = '2.7';
 
     /** Authentication table key. */
-    const ProviderKey = 'Facebook';
+    const PROVIDER_KEY = 'Facebook';
```

Comment:
I did not find any reference to ProviderKey so I'm inclined to accept that change but that's a bit risky.